src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -195,9 +193,9 @@ public partial class EnumConverter : System.ComponentModel.TypeConverter
     }
     public abstract partial class EventDescriptor : System.ComponentModel.MemberDescriptor
     {
-        protected EventDescriptor(System.ComponentModel.MemberDescriptor descr) : base(default(string)) { }
-        protected EventDescriptor(System.ComponentModel.MemberDescriptor descr, System.Attribute[] attrs) : base(default(string)) { }
-        protected EventDescriptor(string name, System.Attribute[] attrs) : base(default(string)) { }
+        protected EventDescriptor(System.ComponentModel.MemberDescriptor descr) : base (default(System.ComponentModel.MemberDescriptor)) { }
```

Comment:
What is the right call to the base class? Running /t:GenerateReferenceSource on the source assembly vs ref assembly gives different results. cc @ericstj, @weshaggard

Reply:
I don't think the base call matters, just that it let the C# compile.